src/PolyType/Abstractions/TypeShapeVisitor.cs

```diff
@@ -13,7 +18,7 @@ public abstract class TypeShapeVisitor : ITypeShapeVisitor
     /// <param name="state">Defines user-provided state.</param>
     /// <returns>The result produced by the visitor.</returns>
     public virtual object? VisitObject<T>(IObjectTypeShape<T> objectShape, object? state = null)
-        => ThrowNotImplementedException(nameof(VisitObject));
+        => ThrowNotImplementedException();
```

Comment:
Would it make sense to add test coverage for the default implementations now, given that this type is moving to the center of the abstractions?

Reply:
Sure. I can do that.